interface/forms/soap/templates/general_new.html

```diff
@@ -4,49 +4,79 @@
 <title>{xl t='SOAP'|escape:'html'}</title>
 
 {headerTemplate}
+{literal}
+ <style type="text/css" title="mystyles" media="all">
+ <!--
```

Comment:
I think adding this to all the forms is very burdensome as it dictates specific styling to the form regardless of the theme selected. My recommendation is to remove all of these.